src/main/java/seedu/address/logic/commands/UnassignCommand.java

```diff
@@ -23,32 +24,33 @@
 public class UnassignCommand extends Command {
     public static final String COMMAND_WORD = "unassign";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes a shift, worker and role assignment from the "
-            + "McScheduler by the index numbers used in the last worker and shift listings. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes shift, worker(s) and role(s) assignment(s) "
+            + "from the McScheduler by the index numbers used in the last worker and shift listings. "
             + "\nParameters: "
             + PREFIX_SHIFT + "SHIFT_INDEX (must be a positive integer) "
             + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD
             + " s/1 "
-            + "w/4";
+            + "w/4 "
+            + "w/2";
 
-    public static final String MESSAGE_UNASSIGN_SUCCESS = "Shift assignment removed:\n%1$s";
+    public static final String MESSAGE_UNASSIGN_SUCCESS = "%d shift assignment(s) removed.";
```

Comment:
Similar to `AssignCommand`, it would be good to list all the assignments removed.